pandas/tests/indexing/test_loc.py

```diff
@@ -1645,6 +1645,21 @@ def test_loc_setitem_mask_td64_series_value(self):
         assert expected == result
         tm.assert_frame_equal(df, df_copy)
 
+    @pytest.mark.parametrize("indexing_func", [list, np.array])
+    @pytest.mark.parametrize("func", ["loc", "iloc"])
+    @pytest.mark.parametrize("rhs_func", [list, np.array])
+    def test_loc_setitem_boolean_list(self, func, rhs_func, indexing_func):
```

Comment:
its only the *-iloc-list cases that fail on master.  at its heart, is this an iloc bug or a loc bug?

Reply:
iloc, thx.